src/Data/Foldable/Compat.hs

```diff
@@ -1,15 +1,37 @@
 {-# OPTIONS_GHC -fno-warn-orphans #-}
 module Data.Foldable.Compat (
   module Base
```

Comment:
Same, here.  Only re-exporting `Base` allows us to use the `.Compat` module with the latest version of base as reference.